src/gui/tray/activitylistmodel.h

```diff
@@ -60,6 +60,7 @@ class ActivityListModel : public QAbstractListModel
         MessageRole,
         DisplayPathRole,
         PathRole,
+        FilePathRole,
```

Comment:
to be honest, I would have an hard time as an user of this model to figure out what I will get when querying the three different roles: `PathRole`, `FilePathRole` and `DisplayPathRole`
given you get a canonical path when we query `FilePathRole` maybe rename it to `CanonicalPathRole` to have an hint at what it is about ?
subsidiary question, should we also use `a._isMultiObjectActivity ?` in other path roles ?

Reply:
In the Qt file info APIs there is a pretty consistent reference to file path being the complete path to a file, and path being the parent folder of said file

I agree that generally this nomenclature is not very clear. We can rename `FilePathRole` to `CanonicalFilePathRole` or `OpenablePathRole`

Regarding `a._isMultiObjectActivity`, we don't need this in other roles -- the main reason we use it here is to differentiate between opening the activity object file directly (when the activity only refers to one object file) or to open the containing folder (when the activity refers to several object files)

Reply:
I changed it to `OpenablePathRole` which is clearer